frontend/src/App.svelte

```diff
@@ -66,7 +69,8 @@
     currentResource,
     rootResource,
     modifierView,
-    bottomLeftPane;
+    bottomLeftPane,
+    selectedComponent;
```

Comment:
Instead of having `selectedComponent` be a variable of `App.svelte`, it should probably be a variable of `ComponentsView.svelte`. Then `ComponentsView` can wrap the other components that make up the view, rather than having `modifierView` be set with one of the `ComponentConfig` views. 

Let me know if this is unclear, and I can elaborate.

Reply:
This makes sense, implemented